packages/core/src/storage/StorageUtility.spec.ts

```diff
@@ -667,4 +672,64 @@ describe("saveSessionInfoToStorage", () => {
       mockedStorage.getForUser("some session", "isLoggedIn", { secure: true })
     ).resolves.toEqual("true");
   });
+
+  let publicKey: KeyLike | undefined;
+  let privateKey: KeyLike | undefined;
+
+  const mockJwk = async (): Promise<{
+    publicKey: KeyLike;
+    privateKey: KeyLike;
+  }> => {
+    if (typeof publicKey === "undefined" || typeof privateKey === "undefined") {
+      const generatedPair = await generateKeyPair("ES256");
+      publicKey = generatedPair.publicKey;
+      privateKey = generatedPair.privateKey;
+    }
+    return {
+      publicKey,
+      privateKey,
+    };
+  };
+
+  const mockKeyPair = async () => {
+    const { privateKey: prvt, publicKey: pblc } = await mockJwk();
+    const dpopKeyPair = {
+      privateKey: prvt,
+      publicKey: await fromKeyLike(pblc),
+    };
+    // The alg property isn't set by fromKeyLike, so set it manually.
+    dpopKeyPair.publicKey.alg = "ES256";
+    return dpopKeyPair;
+  };
+
+  it("saves the DPoP key if provided in the given storage", async () => {
+    const mockedStorage = mockStorageUtility({});
+    const dpopKey = await mockKeyPair();
+    await saveSessionInfoToStorage(
+      mockedStorage,
+      "some session",
+      "an ID token",
+      "https://my.webid",
+      "true",
+      "a refresh token",
+      true,
+      dpopKey
+    );
+
+    expect(
+      JSON.parse(
+        (await mockedStorage.getForUser("some session", "publicKey", {
+          secure: true,
+        }))!
```

Comment:
This appears to be introducing an ESLint warning (see above). Might want to add an exception.

Reply:
c51baf487f7748e2dbe371737f251b197021d9fe